### PR TITLE
Feat/stella rollout

### PR DESCRIPTION
**Pull request by OzgeYurtsever**

### CODE REVIEW

File: src/components/infoMenu/help.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Modal, Button } from "react-bootstrap";
+import http from "../../services/httpService";
+import "./infoMenu.css";
+
+let mode;
+let apiUrl;
+
+class About extends React.Component {
```
Comment (emelalkim) [inline src/components/infoMenu/help.jsx:9]: should class name be changes to Help?

File: src/components/infoMenu/help.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+  render = () => {
+    const mode = sessionStorage.getItem("mode");
+    const feedback = sessionStorage.getItem("feedback");
+    return (
+      // <Modal.Dialog dialogClassName="info-about__modal">
+      <Modal id="modal-fix" show={true}>
+        <Modal.Header className="modal-header">
+          {/* <Modal.Title>Help with {mode === 'teaching' ? "Stella" : "ePAD"} {mode === "lite" ? "Lite" : ""}</Modal.Title> */}
```
Comment (emelalkim) [inline src/components/infoMenu/help.jsx:18]: clean commented out code

File: src/components/infoMenu/help.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    ""
+                  );
+                }}
+              >
+                Get help with ePAD
+              </button>)}
+            {mode === 'teaching' && (<button
+              className="info-about__button"
+              onClick={() => {
+                window.open(
```
Comment (emelalkim) [inline src/components/infoMenu/help.jsx:40]: still opening contact us?

File: src/components/infoMenu/index.jsx [hunk 4]
```diff
@@ -52,12 +53,7 @@ class InfoMenu extends React.Component {
       case "About":
         return <About onOK={this.handleCloseModal} />;
       case "Help":
-        window.open(
-          "https://epad.stanford.edu/documentation/user",
```
Comment (emelalkim) [inline src/components/infoMenu/index.jsx:56]: we talked about moving this to somewhere. I don't see it anywhere in the modified code

File: src/components/infoMenu/help.jsx (hunk 1 above)
Comment (OzgeYurtsever) [reply src/components/infoMenu/help.jsx:9]: Thanks!

File: src/components/infoMenu/help.jsx (hunk 3 above)
Comment (OzgeYurtsever) [reply src/components/infoMenu/help.jsx]: Downloading Stella manual

**Pull request merged**